dev-qt/qtgui/qtgui-5.8.9999.ebuild

```diff
@@ -137,12 +138,14 @@ src_prepare() {
 	use dbus || sed -i -e 's/contains(QT_CONFIG, dbus)/false/' \
 		src/platformsupport/platformsupport.pro || die
 
+	qt_use_disable_config tuio udpsocket src/plugins/generic/generic.pro
```

Comment:
there's a `QT_FOR_CONFIG += gui-private network-private` in that file. I don't know/remember what `QT_FOR_CONFIG` is, so it may not be an issue, but it needs to be investigated.